sale_payment_mgmt/views/account_move_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_account_move_sale_payment_mgmt_form" model="ir.ui.view">
+        <field name="name">account.move.form.sale_payment_mgmt</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <button name="action_invoice_register_payment" position="attributes">
+                <attribute
+                    name="groups"
+                >sales_team.group_sale_salesman,sales_team.group_sale_salesman_all_leads,sales_team.group_sale_manager,account.group_account_invoice</attribute>
```

Comment:
Better to add a new button with your group and same name than to modify existing one.

Reply:
It's a good idea.